lib/ansible/modules/network/eos/eos_static_routes.py

```diff
@@ -0,0 +1,367 @@
+                    description:
+                      - Outgoing interface to take. For anything except 'null0', then next hop IP address should also be configured.
+                      - IP address of the next hop router or
+                      - null0 Null0 interface or
+                      - ethernet e_num Ethernet interface or
+                      - loopback l_num Loopback interface or
+                      - management m_num Management interface or
+                      - port-channel p_num
+                      - vlan v_num
+                      - vxlan vx_num
+                      - Nexthop-Group  Specify nexthop group name
+                      - Tunnel  Tunnel interface
+                      - vtep  Configure VXLAN Tunnel End Points
+                    type: str
+                  nexthop_grp:
+                    description:
+                        - Nexthop group
+                    type: str
+                  admin_distance:
+                    description: 
+                      - Preference or administrative distance of route (range 1-255).
+                    type: int
+                  description:
+                    description:
+                      - Name of the static route.
+                    type: str
+                  tag:
+                    description:
+                      - Route tag value (ranges from 0 to 4294967295).
+                    type: int
+                  track:
+                    description:
+                      - Track value (range 1 - 512). Track must already be configured on the device before adding the route.
+                    type: str
+                  mpls_label:
+                    description:
+                      - MPLS label
+                    type: int
+                  vrf:
+                    description:
+                      - VRF of the destination.
+                    type: str
+  state:
+    description:
+      - The state the configuration should be left in.
+    type: str
+    choices:
+      ['deleted', 'merged', 'overridden', 'replaced', 'gathered', 'rendered']
+    default:
+      merged
+"""
+EXAMPLES = """
+# Using deleted
+
+Before State
+-------------
+veos(config)#show running-config | grep "route"
+ip route 165.10.1.0/24 Ethernet1 100
+ip route 172.17.252.0/24 Nexthop-Group testgroup
+ip route vrf testvrf 130.1.122.0/24 Ethernet1 tag 50
+ipv6 route 5001::/64 Ethernet1 50
+veos(config)#
+
+- name: Delete static route configuration
+  eos_static_routes:
+    state: deleted
+
+After State
+-----------
+
+veos(config)#show running-config | grep "route"
+veos(config)#
+
+
+# Using merged
+
+Before State
+-------------
+veos(config)#show running-config | grep "route"
+ip route 165.10.1.0/24 Ethernet1 100
+ip route 172.17.252.0/24 Nexthop-Group testgroup
+ip route vrf testvrf 130.1.122.0/24 Ethernet1 tag 50
+ipv6 route 5001::/64 Ethernet1 50
+veos(config)#
+
+- name: Merge new static route configuration
```

Comment:
we can also show before and after lists with set of commands which is actual output ? though I am not sure about how others have included this .